node/src/chain/acceptor.rs

```diff
@@ -459,7 +481,14 @@ async fn verify_block_cert(
     )
     .await
     {
-        return Err(anyhow!("invalid first reduction votes"));
+        return Err(anyhow!(
+            "invalid step votes, 1st reduction, hash = {}, round = {}, iter = {}, seed = {},  sv = {:?} ",
+            to_str(&hdr.block_hash),
+            hdr.round,
+            iteration,
+            to_str(&curr_seed.inner()),
+            cert.first_reduction
+        ));
```

Comment:
Should you propagate the real error here? There is no usage of the previously exported `if let Err(e) = ...`

Reply:
Good point